dashboard/app/models/user.rb

```diff
@@ -958,6 +958,27 @@ def student_of?(teacher)
     teachers.include? teacher
   end
 
+  # @return {boolean} true if (1) Attended CSD TeacherCon '17 (2) are a CSD facilitator
+  def circuit_playground_pd_eligible?
+    csd_cohorts = %w(CSD-TeacherConPhiladelphia CSD-TeacherConPhoenix CSD-TeacherConHouston)
```

Comment:
Again, I don't love hard coding cohort names like this, but I'm not sure there's a better approach.